examples/poyo/train.py

```diff
@@ -287,10 +285,11 @@ def main(cfg: DictConfig):
         )
 
     # get modality details
-    readout_spec = MODALITIY_REGISTRY[cfg.readout_id]
+    readout_id = cfg.dataset[0].config.readout.readout_id
+    readout_spec = MODALITIY_REGISTRY[readout_id]
```

Comment:
Can you add a TODO comment that we need to make sure the readout_ids are the same for all datasets.